go.sum

```diff
@@ -29,6 +29,7 @@ github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc/go.mod h1:LOuy
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
```

Comment:
Can you move tools.go in your project to a separate directory with a separate go.mod file? Because otherwise your linter [depends on golangci-lint package](https://github.com/kunwardeep/paralleltest/blob/master/go.mod#L6) and it brings a lot of obsolete packages here because of a circle dependency:
```
golangci-lint@latest -> paralleltest@1.0.1 -> golangci-lint@v1.31.0
```

Reply:
done 👍